src/app/features/auth/pages/login/login-page.component.html

```diff
@@ -1,8 +1,5 @@
 <spinner *ngIf="loading$ | async" size="lg" />
 <div class="bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4 form-middle">
-  <div class="mb-4 flex gap-2">
-    <img alt="" src="assets/icons/Logo.svg" />
-    <h1>Freenance</h1>
-  </div>
+  <logo title="Freenance" src="assets/icons/Logo.svg" alt="Freenance logo" />
```

Comment:
Not enough level of abstraction. I'd create a whole new Page component to have all the common styles and elements for both log in and register